Script/AtomicEditor/editor/Preferences.ts

```diff
@@ -373,7 +375,8 @@ class PreferencesFormat {
             width: 0,
             height: 0,
             monitor: 0,
-            maximized: false
+            maximized: false,
+            windowed: false
```

Comment:
Does this default the player and editor to fullscreen? Where are these being used?